daprdocs/content/en/dotnet-sdk-docs/dotnet-client/_index.md

```diff
@@ -199,6 +199,63 @@ await foreach (var items in subscribeConfigurationResponse.Source.WithCancellati
 }
 ```
 
+### Distributed lock (Alpha)
+
+#### Acquire a lock
+
+```csharp
+using System;
+using Dapr.Client;
+
+namespace LockService
+{
+    class Program
+    {
+        [Obsolete("Distributed Lock API is in Alpha, this can be removed once it is stable.")]
+        static async Task Main(string[] args)
+        {
+            string DAPR_LOCK_NAME = "lockstore";
+            string fileName = "my_file_name";
+            var client = new DaprClientBuilder().Build();
+    
+            await using (var fileLock = await client.Lock(DAPR_LOCK_NAME, fileName, "random_id_abc123", 60))
```

Comment:
It's worth calling out here in a comment that locking it this way will also unlock it automatically since this is a disposable object.